
Failures (3):
  JUnit Vintage:Specialfilter_163_1Test:testSpecialfilterWithNumbersGreaterThan10AndSpecial
    MethodSource [className = 'original.Specialfilter_163_1Test', methodName = 'testSpecialfilterWithNumbersGreaterThan10AndSpecial', methodParameterTypes = '']
    => java.lang.AssertionError: expected:<1> but was:<2>
       org.junit.Assert.fail(Assert.java:89)
       org.junit.Assert.failNotEquals(Assert.java:835)
       org.junit.Assert.assertEquals(Assert.java:647)
       org.junit.Assert.assertEquals(Assert.java:633)
       original.Specialfilter_163_1Test.testSpecialfilterWithNumbersGreaterThan10AndSpecial(Specialfilter_163_1Test.java:44)
       java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
       java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
       java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
       java.base/java.lang.reflect.Method.invoke(Method.java:566)
       org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
       [...]
  JUnit Vintage:Specialfilter_163_1Test:testSpecialfilterWithNumbersGreaterThan10ButNotSpecial
    MethodSource [className = 'original.Specialfilter_163_1Test', methodName = 'testSpecialfilterWithNumbersGreaterThan10ButNotSpecial', methodParameterTypes = '']
    => java.lang.AssertionError: expected:<0> but was:<1>
       org.junit.Assert.fail(Assert.java:89)
       org.junit.Assert.failNotEquals(Assert.java:835)
       org.junit.Assert.assertEquals(Assert.java:647)
       org.junit.Assert.assertEquals(Assert.java:633)
       original.Specialfilter_163_1Test.testSpecialfilterWithNumbersGreaterThan10ButNotSpecial(Specialfilter_163_1Test.java:37)
       java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
       java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
       java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
       java.base/java.lang.reflect.Method.invoke(Method.java:566)
       org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
       [...]
  JUnit Vintage:Specialfilter_163_1Test:testSpecialfilter
    MethodSource [className = 'original.Specialfilter_163_1Test', methodName = 'testSpecialfilter', methodParameterTypes = '']
    => java.lang.AssertionError: expected:<3> but was:<1>
       org.junit.Assert.fail(Assert.java:89)
       org.junit.Assert.failNotEquals(Assert.java:835)
       org.junit.Assert.assertEquals(Assert.java:647)
       org.junit.Assert.assertEquals(Assert.java:633)
       original.Specialfilter_163_1Test.testSpecialfilter(Specialfilter_163_1Test.java:16)
       java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
       java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
       java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
       java.base/java.lang.reflect.Method.invoke(Method.java:566)
       org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
       [...]

Test run finished after 47 ms
[         4 containers found      ]
[         0 containers skipped    ]
[         4 containers started    ]
[         0 containers aborted    ]
[         4 containers successful ]
[         0 containers failed     ]
[         5 tests found           ]
[         0 tests skipped         ]
[         5 tests started         ]
[         0 tests aborted         ]
[         2 tests successful      ]
[         3 tests failed          ]

